lego/apps/users/models.py

```diff
@@ -377,6 +377,17 @@ def get_default_picture(self):
         else:
             return "default_other_avatar.png"
 
+    def delete(self, using=None, force=False):
+        from lego.apps.events import constants
+        from lego.apps.events.models import Event
+
+        if force:
+            for event in Event.objects.all():
+                if event.is_admitted(self):
+                    if event.registrations.get(user=self).presence == constants.PRESENT:
```

Comment:
Again, you could do `event.registrations.filter(user=self, presence=constants.PRESENT).exists()` to avoid db lookups. (not really important here though)

Reply:
Is this still relevant to do in latest commmit?

Reply:
nope. Looks good!